yaml2ics.py

```diff
@@ -90,6 +90,15 @@ def event_from_yaml(event_yaml: dict, tz: tzinfo = None) -> ics.Event:
             )
         )
 
+        if "except_on" in repeat:
+            for exdate in repeat.get("except_on"):
+                event.extra.append(
+                    ics.ContentLine(
```

Comment:
Should this not be combined into one line? The [example from the RFC](https://www.kanzaki.com/docs/ical/exdate.html) is:

```
  EXDATE:19960402T010000Z,19960403T010000Z,19960404T010000Z
```